src/style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :last-child {
+  margin: 0;
+}
```

Comment:
you can use your `margin-left: 4px;` with `:not(:first-child)` instead of the above